internal/auth/telemetry_claim.go

```diff
@@ -51,3 +53,36 @@ func getTelemetryClaim(ctx context.Context) (*TelemetryClaim, error) {
 	}
 	return claim, nil
 }
+
+func ValidAttestationClaim(ctx context.Context, filter *model.AttestationFilter) bool {
```

Comment:
I feel like this approach is less code, which could be good. All good.

Reply:
agreed that less code is good!